boot/makebootable.go

```diff
@@ -312,6 +312,25 @@ type makeRunnableOptions struct {
 	AfterDataReset bool
 }
 
+func copyNoSymLink(orig, dst string) error {
```

Comment:
this probably needs a different name/signature otherwise the comment in it is a bit odd

Reply:
changed to copyBootSnap